recipes/FC-Virus/mete.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: FC-Virus
+  version: "1.0.0"
+  noarch: generic
+
+source:
+  url: https://github.com/qdu-bioinfo/FC-Virus/archive/refs/heads/main.tar.gz
+  git_tag: main
```

Comment:
Please remove this!
It is ignored now anyway. It should be used with 'git_url'

Reply:
Thanks for your reply. I have removed that section as you suggested.